docs/get-started/v2/get-started/prebuilt/prebuilt.mdx

```diff
@@ -75,26 +78,27 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
     src="https://<your-template-subdomain>/meeting/<room-code>"
     style="height: 100vh; width: 100%; border:0;"
 >
+
+
 </iframe>
 ```
 
 </StepsContainer>
 
 ## Embed as Component
 
-100ms Prebuilt can be embedded as a native component `HMSPrebuilt` by importing HMSRoomKit in your apps.
+100ms Prebuilt can be embedded as a native component `HMSPrebuilt` by importing `RoomKit` package in your app.
 
 <FlexContainer>
 <Callout icon="link" title="HMSPrebuilt Component Basics">
+    - **RoomKit** -  Collection of embeddable 100ms UI components for your app.
+        - `HMSPrebuilt` is the first embeddable UI component of RoomKit package.
+            - Have a Component request? Drop a note here
```

Comment:
"Drop a note here" - where?

Reply:
wil be asking @akankshat100ms  to add it in base branch